README.md

```diff
@@ -194,9 +195,50 @@ $ solid start --help
     --email-auth-pass [value]     Password of your email service
     --useApiApps                  Do you want to load your default apps on /api/apps?
     --api-apps [value]            Path to the folder to mount on /api/apps
+    --redirect-http-from [value]  HTTP port or ','-separated ports to redirect to the solid server port (e.g. "80,8080").
     -v, --verbose                 Print the logs to console
+    -h, --help                    output usage information
  ```
 
+Instead of using flags, these same options can also be configured via the following environment variables:
```

Comment:
What takes precedence?

Reply:
Also, there's now 3 options: config, flags, env. Explain why 3 and why one would prefer any of these in what situations.